frontend/src/it/scala/bloop/CommunityBuild.scala

```diff
@@ -1,34 +1,21 @@
 package bloop
 
-import bloop.io.AbsolutePath
-import java.nio.file.Path
-import java.nio.file.Files
 import java.nio.charset.StandardCharsets
-import bloop.logging.Logger
-import bloop.engine.BuildLoader
-import bloop.engine.Build
-import bloop.engine.State
-import sbt.internal.inc.bloop.ZincInternals
-import bloop.logging.NoopLogger
-import bloop.engine.caches.ResultsCache
-import bloop.logging.BloopLogger
-import bloop.data.Project
-import bloop.config.Config
-import bloop.data.Origin
+import java.nio.file.{Files, Path}
 import java.nio.file.attribute.FileTime
-import bloop.engine.Dag
-import bloop.engine.Exit
-import bloop.cli.ExitStatus
-import bloop.engine.Run
-import bloop.cli.Commands
-import bloop.engine.Action
+import scala.concurrent.Await
 import scala.concurrent.duration.Duration
-import bloop.engine.ExecutionContext
-import bloop.engine.Interpreter
+import bloop.cli.{Commands, ExitStatus}
+import bloop.config.Config
+import bloop.data.{Origin, Project}
+import bloop.engine._
```

Comment:
Let's try to avoid wildcard imports as much as possible and instead prefer explicit import :smile: I suppose it was your IntelliJ configuration the one that did this change, so you might want to disable it when you work in the bloop codebase.